src/main/pages/che-7/installation-guide/proc_gerenating-self-signed-certificates.adoc

```diff
@@ -0,0 +1,125 @@
+[id="gerenating-self-signed-certificates_{context}"]
+= Generating self-signed TLS certificates
+
+This section describes how to prepare self-signed TLS certificates to use with {prod-short} on different platforms.
+
+
+[discrete]
+== Prerequisites
+
+* Find out expected domain name where {prod-short} deployment is planned
+
++
+[subs="+quotes"]
+----
+# default for minishift
+$( minishift ip ).nip.io
+
+# default for minikube
+$( minikube ip ).nip.io
+
+# default for crc
+apps-crc.testing
+----
+
+* Find out `openssl.cnf` file location on target machine
+
++
+[subs="+quotes"]
+----
+# Fedora 31
+/etc/pki/tls/openssl.cnf
+----
+
+[discrete]
+== Procedure
+
+. Set necessary environment variables
+
++
+[subs="+quotes"]
+----
+$ CA_CN=eclipse-che-signer
+$ DOMAIN=*.<expected.domain.com>
+$ OPENSSL_CNF=<path_to_openssl.cnf>
+----
+
++
+[subs="+quotes"]
+----
+# Example with minikube on Fedora 31
+$ CA_CN=eclipse-che-signer
+$ DOMAIN=*.$( minikube ip ).nip.io
+$ OPENSSL_CNF=/etc/pki/tls/openssl.cnf
+----
+
```

Comment:
OSX example

```bash
export CA_CN=eclipse-che-signer
export DOMAIN=*.apps-crc.testing
export OPENSSL_CNF=/System/Library/OpenSSL/openssl.cnf
```

Reply:
I've added it as second example https://github.com/eclipse/che-docs/pull/1007/files#diff-996f569c48ab17cabef910cc083fbae8R55